src/test/java/com/openlattice/rehearsal/data/DataControllerTest.kt

```diff
@@ -344,7 +344,7 @@ class DataControllerTest : MultipleAuthenticatedUsersBase() {
             Assert.fail("Should have thrown Exception but did not!")
         } catch (e: UndeclaredThrowableException) {
             Assert.assertTrue(e.undeclaredThrowable.message!!
-                    .contains("differs from allowed entity types in association type ${edge1.id}", true))
+                    .contains("differs from allowed entity types ([]) in association type of entity set ${esEdge1.id}", true))
```

Comment:
Not sure what the right way to do this is, but I feel like at some point we should avoid matching on the exact error string to allow for flexibility in the messages. But it does seem valuable to be able to compare the specific ids / fields that are causing the errors.